src/Pos/Communication/Limits.hs

```diff
@@ -209,9 +228,9 @@ instance MessageLimited (DataMsg GtMsgContents) where
     getMsgLenLimit _ =
         return $ each %~ fmap DataMsg $
             ( mcCommitmentMsgLenLimit
-            , noLimit
+            , NoLimit
```

Comment:
1. Do you mean that it contains `AsBinary Secret`? AFAIK all `instance Bi (AsBinary a)` are limited.
2.1. My motivation was the following: if the one changes type, he would most probably forget to change limit if it's just a magic constant. When he is going to change it, he would need to evaluate length somehow, which is unpleasant for complex types. And now it's evaluated automatically
2.2. Actually I relied on tests to not allow me to write invalid limit. But probably I shouldn't. Then why a "value such that we can easily parse so many bytes and at the same time it can fit all fixed-size data" was bad?